src/Tools/Radiation/Models/Cone/Emission/Bremsstrahlung/ConeBremsstrahlungScreenedEmission.cpp

```diff
@@ -60,9 +59,8 @@ void ConeBremsstrahlungScreenedEmission::CalculateTotalEmission() {
  * Calculate eq. 4BS (Koch & Motz) for a given species
  */
 slibreal_t ConeBremsstrahlungScreenedEmission::Calculate4BS(slibreal_t Z) {
-    slibreal_t Z2fakt = 4*Z*Z*r02Alpha;
+    slibreal_t Z2fakt = 4*Z*Z*r02/137;
```

Comment:
Not super important, but an interesting point to note from Koch & Motz: In their paper, 1/137 is actually not as much a numeric value as a mnemonic for the fine structure constant. So when they write 1/137, they actually mean ``1/137.035999...``